src/commands/stream.cr

```diff
@@ -87,10 +87,10 @@ module Redis::Commands::Stream
 
   # Return the entries in the given stream between the `start` and `end` ids.
   # If `count` is provided, Redis will return only that number of entries.
-  def xrange(key : String, start min, end max, count = nil)
+  def xrange(key : String, start min, end max, count : String | Int | Nil = nil)
```

Comment:
While we're here, since stream ids are strings, we can add types to `start` and `end` for good measure. It'll still fail at compile time without them, but this will improve the error message the developer sees. I'll go ahead and merge after that.

```suggestion
  def xrange(key : String, start min : String, end max : String, count : String | Int | Nil = nil)
```